docs/markdown/Cross-compilation.md

```diff
@@ -44,11 +49,25 @@ compiler that runs on Windows but produces binaries on MIPS Linux. In
 this case *build machine* is x86 Linux, *host machine* is x86 Windows
 and *target machine* is MIPS Linux. This setup is known as the
 [Canadian
-Cross](https://en.wikipedia.org/wiki/Cross_compiler#Canadian_Cross). As
-a side note, be careful when reading cross compilation articles on
-Wikipedia or the net in general. It is very common for them to get
+Cross](https://en.wikipedia.org/wiki/Cross_compiler#Canadian_Cross).
+\[As a side note, be careful when reading cross compilation articles
```

Comment:
Let's not have the brackets here. If this needs changing then let's rewrite the text itself to flow better.

Reply:
Took those out, and tightened up the "target machine" bullet above so there was no temptation.